docs/_static/mxnet-theme/index.html

```diff
@@ -17,27 +17,26 @@
     </div>
   </div>
 </div>
-
 <div class="section-tout">
   <div class="container">
     <div class="row">
       <div class="col-lg-4 col-sm-12">
-        <h3>Apache MXNet 1.0 Released</h3>
-        <p>We're excited to announce the release of MXNet 1.0! Check out the release notes for latest updates.</p>
-        <a href="https://github.com/apache/incubator-mxnet/releases/tag/1.0.0">Learn More</a>
+        <h3>Apache MXNet 1.0.1 Released</h3>
+        <p>We're excited to announce the release of MXNet 1.0.1! Check out the release notes for latest updates.</p>
+        <a href="https://github.com/apache/incubator-mxnet/releases/tag/1.0.1">Learn More</a>
```

Comment:
@mbaijal are we supposed to have `1.1.0.rc0` in the link or `1.1.0`?

Reply:
It should be `1.1.0` everywhere. Once the RC is created, no src can be modified so it must be the final release tag.